### PR TITLE
Public attestation endpoint

### PR DESCRIPTION
**Pull request by eccles**
Solution:
Access to the public assets endpoint for attestation.

Signed-off-by: Paul Hewlett <phewlett76@gmail.com>

### CODE REVIEW

File: archivist/constants.py [hunk 1]
```diff
@@ -30,6 +30,10 @@
 ASSETS_WILDCARD = "assets/-"
 EVENTS_LABEL = "events"
 
+PUBLICASSETS_SUBPATH = "v2"
+PUBLICASSETS_LABEL = "publicassets"
+PUBLICASSETS_WILDCARD = "publicassets/-"
```
Comment (wp0pw) [inline archivist/constants.py:35]: I don't think we need that one?

File: archivist/publicevents.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+            props (dict): optional properties e.g. {"confirmation_status": "CONFIRMED" }
+            attrs (dict): optional attributes e.g. {"arc_display_type": "open" }
+            asset_attrs (dict): optional asset_attributes e.g. {"arc_display_type": "door" }
+
+        Returns:
+            integer count of assets.
+
+        """
+
+        asset_id = asset_id or PUBLICASSETS_WILDCARD
```
Comment (wp0pw) [inline archivist/publicevents.py:125]: yeh, don't think the wildcard is allowed here

File: archivist/publicevents.py [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+            props (dict): e.g. {"tracked": "TRACKED" }
+            attrs (dict): e.g. {"arc_display_type": "open" }
+            asset_attrs (dict): optional asset_attributes e.g. {"arc_display_type": "door" }
+            page_size (int): optional page size. (Rarely used).
+
+        Returns:
+            iterable that returns :class:`Event` instances
+
+        """
+        asset_id = asset_id or PUBLICASSETS_WILDCARD
```
Comment (wp0pw) [inline archivist/publicevents.py:155]: as above, no wildcard

Comment (eccles) [reply archivist/publicevents.py]: removed it

File: archivist/publicevents.py (hunk 2 above)
Comment (eccles) [reply archivist/publicevents.py]: removed

**Review by wp0pw — APPROVED** [review]
LGTM

**Pull request merged**